tests/test_mdns_service.js

```diff
@@ -0,0 +1,29 @@
+var dns_sd = require('../lib/dns_sd')
+  , proxyquire =  require('proxyquire')
```

Comment:
More proxyquire business. Here it is required. But it doesn't appear in the `devDependencies` in package.json. Is it installed globally on your system?